package.json

```diff
@@ -40,7 +40,7 @@
     "gulp-streamify": "^1.0.2",
     "gulp-uglify": "^1.5.3",
     "gulp-util": "^3.0.7",
-    "jasmine-core": "^2.4.1",
+    "jasmine-core": "2.5.2",
```

Comment:
Might be worth bumping `karma` related packages because we seemed to be horribly behind.

See https://github.com/mavenlink/mavenlink/commit/643c80810c818d9b911fcd27fdb665f7977954cc for where we're at in bigmaven.